src/factory/DecoratorFactory.ts

```diff
@@ -43,8 +43,8 @@ export class InternalDecoratorFactory {
       return (target: Object, name: string, _descriptor?: PropertyDescriptor): PropertyDescriptor => {
         const descriptor = this._resolveDescriptor(target, name, _descriptor);
         const { value, writable, enumerable, configurable, get, set } = descriptor;
-        const isFirstInstance = !InstanceChainMap.has([ target, name ]);
-        const chainData = InstanceChainMap.get([ target, name ]) || { fns: [], properties: [] };
+        const isFirstInstance = !InstanceChainMap.has([target, name]);
```

Comment:
Can you rollback any formatting that was autoformatted to reduce the scope of changes?